pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -230,6 +230,9 @@ type LogCollectorSpec struct {
 	// Periodicity is the periodicity of the log rotation
 	// +optional
 	Periodicity string `json:"periodicity,omitempty"`
+	// MaxLogSize is the maximum size of the log in GiB
+	// +optional
+	MaxLogSize int `json:"maxLogSize"`
```

Comment:
I'll  change the script accordingly

Reply:
IMO... We should be sure to be explicit here, this is **per file** and not for the whole log directory. We should provide users the information they need to do the calculations to understand what their disk usage can be. And we should also be clear that the logs can grow larger than this value because we only check every 30 minutes.